aptos-move/framework/cached-packages/src/aptos_framework_sdk_builder.rs

```diff
@@ -131,11 +131,6 @@ pub enum EntryFunctionCall {
         coin_type: TypeTag,
     },
```

Comment:
Why do we change this file?

Reply:
See discussion in #move, a previous PR should have checked these in but somehow it got through without.